situp-plugins/elasticsearch/src/main/java/com/amazon/situp/plugins/sink/elasticsearch/BackOffUtils.java

```diff
@@ -0,0 +1,30 @@
+package com.amazon.situp.plugins.sink.elasticsearch;
+
+import org.elasticsearch.common.unit.TimeValue;
+
+import java.util.Iterator;
+
+public final class BackOffUtils {
```

Comment:
can we not make the initialAttempt as a part of this Util?

Reply:
Checking...